.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This should probably extend eslint's recommended settings (`eslint:recommended`).